### PR TITLE
Convert carousel css to tailwindcss

### PR DESCRIPTION
**Pull request by Karmotrine**
- convert carousel css to tailwindcss
- remove unused module.css (about, hero-carousel, page)
- add mockdataImage.ts for carousel mock data
- add "picsum.photos" for image domain

### CODE REVIEW

File: src/mock/mockdataImage.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+interface MockImage {
+    id: number;
```
Comment (Lemon1903) [inline src/mock/mockdataImage.ts:2]: `id` not used

File: src/components/Home/Hero/ShowcaseCarousel.tsx [hunk 2]
```diff
@@ -1,9 +1,13 @@
-import "@/styles/hero-carousel.module.css";
 import { ArrowDown } from "@phosphor-icons/react";
 import "keen-slider/keen-slider.min.css";
 import { useKeenSlider } from "keen-slider/react";
+import { mockDataImage } from "@/mock/mockdataImage";
+import Image from "next/image";
 
 export default function ShowcaseCarousel() {
+
+  const heroSlider = "flex items-center justify-center text-[50px] text-white font-medium min-w-[459.35px] min-h-[424.48px] max-h-[459.35px]"
```
Comment (Lemon1903) [inline src/components/Home/Hero/ShowcaseCarousel.tsx:9]: I think this should be placed directly on the HTML element it should contain. Having one long line with tailwind is inevitable I guess if it really have many styles in it.

File: src/components/Home/News/index.tsx [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
-import "keen-slider/keen-slider.min.css";
 import { useKeenSlider } from "keen-slider/react";
+import Image from "next/image";
+import { mockDataImage } from "@/mock/mockdataImage";
 
 //https://codesandbox.io/s/github/rcbyr/keen-slider-sandboxes/tree/v6/basic/free-mode/react-typescript?file=/src/styles.css
 
 export default function HomeNews() {
+
+  const numberSlide = "flex items-center justify-center text-[50px] text-black font-medium min-h-[271px] min-w-[339px] max-h-[339px]"
```
Comment (Lemon1903) [inline src/components/Home/News/index.tsx:10]: Same reason above

**Comment by Lemon1903** [comment]
The code is running well and the carousels is working well already. My question is that the `ShowcaseCarousel` really looks like this? Or should it fill the whole container?
image

I have concerns with too much arbitrary values, is it temporary only or is it what it is already? Is it possible to use the tailwind scale at most cases? Like use the closest scale to the arbitrary value?

Lastly, just code cleanup, formatting, and comments of parts for better readability and code quality.

**Review by Lemon1903 — COMMENTED** [review]
The code is running well and the carousels are working well already. My question is that the `ShowcaseCarousel` really looks like this? Or should it fill the whole container?
image

I have concerns with too much arbitrary values, is it temporary only or is it what it is already? Is it possible to use the tailwind scale at most cases? Like use the closest scale to the arbitrary value?

Lastly, just code cleanup, formatting, and comments of parts for better readability and code quality.

**Comment by Karmotrine** [comment]
> The code is running well and the carousels are working well already. My question is that the `ShowcaseCarousel` really looks like this? Or should it fill the whole container? image
> 
> I have concerns with too much arbitrary values, is it temporary only or is it what it is already? Is it possible to use the tailwind scale at most cases? Like use the closest scale to the arbitrary value?
> 
> Lastly, just code cleanup, formatting, and comments of parts for better readability and code quality.

Based every value on Figma mockups, scales could work too but what we have so far is low fidelity designs, that's why I did not put much concern on width/height values.
<hr>

> I think this should be placed directly on the HTML element it should contain. Having one long line with tailwind is inevitable I guess if it really have many styles in it.

> `id` not used

Will refactor these

**Comment by Karmotrine** [comment]
> The code is running well and the carousels are working well already. My question is that the `ShowcaseCarousel` really looks like this? Or should it fill the whole container? image

This can be worked on a separate branch feature. Something to do with `perView`  param of each carousel component.

**Review by Lemon1903 — CHANGES_REQUESTED** [review]
> Based every value on Figma mockups, scales could work too but what we have so far is low fidelity designs, that's why I did not put much concern on width/height values.

I see, so these values are temporary for now and will be changed when there are high fidelity wireframes already.

I have found no bugs from the code and the carousel is working perfectly already and the output looks good to me already. My only concern for now would be the in code formatting for code consistency like arrangement of tailwind utilities which can be done quickly with one prettier format, using template literals for `className` which acts like a normal string, and using arbitrary value even if there is a corresponding tailwind scale to it.

**Comment by Karmotrine** [comment]
Repository can now be prettified by one command: `npm run prettify`
Template literals is not really possible especially in `.tsx` component props since it needs to be enclosed within curly braces.
Using Scales (using fractions) is usually dependent on zoom size

**Comment by Lemon1903** [comment]
The sizes of everything could be adjusted once high fidelity comes out and should not be focused for now. Then cleaning up the unecessary template literals on classnames could be fixed on another branch. Alright looks good to me!

**Review by Lemon1903 — APPROVED** [review]
The sizes of everything could be adjusted once high fidelity comes out and should not be focused for now. Then cleaning up the unecessary template literals on classnames could be fixed on another branch. Alright looks good to me!

**Pull request merged**